docs/providers/google/events/event.md

```diff
@@ -27,8 +27,9 @@ functions:
     events:
       - event:
           eventType: providers/cloud.pubsub/eventTypes/topic.publish
-          resource: projects/*/topics/my-topic
+          resource: projects/project/topics/my-topic
```

Comment:
I think referencing `project` would be error-prone.

```suggestion
          resource: 'projects/${self:provider.project, ""}/topics/my-topic'
```

Reply:
@exoego great point. Just updated the commit to include your suggested change, which makes the added note no longer necessary.